app/letters/rest.py

```diff
@@ -14,6 +12,4 @@
 def create_process_returned_letters_job():
```

Comment:
Should this whole method be removed?

Reply:
yes it should, but then the tests needs to be removed. So following the principle I was using below - I didn't remove it.